pages/linux/xz.md

```diff
@@ -0,0 +1,30 @@
+# xz
+
+> Data compression software with a high compression ratio.
+> Can compress typical files ~30% smaller than gzip and ~15% smaller output than bzip2.
+> Syntax is similar to that of gzip.
+> More information: <https://tukaani.org/xz/>.
```

Comment:
I couldn't find the documentation at this link. Can you update this one with a more specific one?

Reply:
Hi @kbdharun I couldn't find any documentation provided by them directly, most of this info was grabbed from man pages directly. If you want I can update with a link to a man page site as that's probably more useful to users
https://manpages.org/xz

Reply:
> Hi @kbdharun I couldn't find any documentation provided by them directly, most of this info was grabbed from man pages directly. If you want I can update with a link to a man page site as that's probably more useful to users https://manpages.org/xz

That would be great, we generally prefer manned for man pages.

Reply:
Will do, Ill hold off on the change until the other comments are confirmed and get them all submitted together. Thanks!